vms/platformvm/txs/codec.go

```diff
@@ -115,6 +115,7 @@ func RegisterUnsignedTxsTypes(targetCodec codec.CaminoRegistry) error {
 		targetCodec.RegisterCustomType(&RegisterNodeTx{}),
 		targetCodec.RegisterCustomType(&BaseTx{}),
 		targetCodec.RegisterCustomType(&MultisigAliasTx{}),
+		targetCodec.RegisterCustomType(&SetValidatorStateTx{}),
```

Comment:
plz add it to the end, in case someone already using existing code types order (magellan, caminojs)